packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1760,6 +1761,13 @@ def test_3dt_bundle_download_and_extract(cluster):
                 assert 'ip' in health_report
                 assert health_report['ip'] == master_ip
 
+                # make sure systemd unit output is correct and does not contain error message
+                gzipped_unit_output = z.open(master_folder + 'dcos-mesos-master.service.gz')
+                unit_output = gzip.decompress(gzipped_unit_output.read())
+                assert ('Hint: You are currently not seeing messages from '
```

Comment:
This seems like a really fragile string to be searching for. Works for now, but would be nice if there was some definite thing we could hook onto. If we start localizing / internationalizing (some discussions going around), will be necessary.